firefox-ios/Client/Application/ImageIdentifiers.swift

```diff
@@ -62,17 +62,13 @@ public struct ImageIdentifiers {
         }
     }
 
-    public struct TrackingProtection {
-        public static let protectionAlert = "protectionAlert"
-        public static let protectionOn = "protectionOn"
-        public static let protectionOff = "protectionOff"
-
-        public static let socialMediaTrackers = "socialMediaTrackersImage"
-        public static let fingerprintersTrackers = "fingerprintersTrackersImage"
-        public static let crossSiteTrackers = "crossSiteTrackersImage"
-    }
-
     public struct NativeErrorPage {
         public static let noInternetConnection = "noInternetConnection"
     }
+
+    public struct TrackingProtection {
+        public static let protectionAlert = "protectionAlertLarge"
```

Comment:
These images are not standard images. As such naming the images with the suffix `Large` is confusing as the usually standard images have the size suffix. I think these images should be named closer to their file names (e.g. `protectionPanelOn`).